### ISSUE TITLE
Can we use the bounding box in the  testing set?

### ISSUE THREAD

**Issue by SaulZhang** [liuziwei7/fashion-landmarks#10]
Dear Ziwei Liu:
       A problem confuses me.Can we use the bounding box position as  a konwn information in the testing set. I notice that you trian three network for upper,lower and full-body clothes. As I can not find the train code. Do you use the bounding box information to pre-process the training data.(crop the bounding box and then padding or resizing to 224*224?).If so,should we train a network to catch the bounding box position?(each class having an object detection network for the bounding box coordinate so that we can test a image and crop the information from the bounding box?).But the object detection network seems difficult to train because we can find that some images both having the upper clothing information and the lower clothing information, how can the network resolve  what is need to recognition?Thanks for your answer.

**Comment by liuziwei7** [comment]
Actually we assume the ground truth bounding box is given during testing in this work. Thus, the input to the "Deep Fashion Alignment" system is a cropped clothing item.

**Comment by SaulZhang** [comment]
Thanks for your answer.
             In These days,I also learned about your another paper《Unconstrained Fashion Landmark Detection via Hierarchical Recurrent Transformer Networks》.In this paper you  proposed the DLAN which solving the problem about the provided bounding box.But I could not find the corresponding code in your github repositories.It seems thatThe page has been lost.Can you give me a link about the DLAN code.

**Comment by liuziwei7** [comment]
Please contact Dr. Sijie Yan (yysijie@gmail.com) for the code and models of "Unconstrained Fashion Landmark Detection via Hierarchical Recurrent Transformer Networks".
